docs/user-guide/global-configurations/deployment-window.md

```diff
@@ -0,0 +1,117 @@
+| **In case of overlap?** | Blackout window gets a higher priority over maintenance window | Maintenance window has a lower priority |
+
+
+### Difference between a Blackout Window and Maintenance Window
+
+Technically, both of them are different methods of restricting deployments to an environment. For example, specifying either a blackout window of [8:00 AM to 10:00 PM] or a maintenance window of [10:00 PM to 8:00 AM] essentially does the same job. You can define either of them depending on your use case.
+
+---
+
+## Configuring Deployment Window
+
+(This is configured by super-admins. Details yet to be added.)
+
+---
+
+## Checking Deployment Window
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have view-only permission or above (along with access to the environment and application) to check deployment windows.
+{% endhint %}
+
+### Overview Page
+
+The **Deployment window** section shows the deployment windows configured for each [environment](../../reference/glossary.md#environment).
+
+![Figure 1: Overview Page - Deployment Window](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/overview-deployment-win.jpg)
+
+However, if a deployment window doesn't exist for an environment, the message `No deployment windows are configured` would be displayed next to it.
+
+You may click the dropdown icon to view the details which include:
+* Type of deployment window (Blackout/Maintenance)
+* Name and description
+* Frequency of window (once, weekly, monthly, yearly)
+* Start date and time 
+* End date and time
+
+### App Details Page
+
+Unlike the **Overview** page which shows deployment windows for all environments, the **App Details** page shows the deployment windows for one environment at a time.
+
+![Figure 1: App Details Page - Deployment Window](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/app-details-deployment-win.jpg)
+
+If configured, the ongoing and upcoming deployment windows would be visible in the form of cards (in the same row that has `Application Status`, `Deployment Status`, etc.)
+
+For example, if the super-admin has configured 4 deployment windows (say 2 Blackout and 2 Maintenance), you will see 4 cards stacked upon each other. However, no cards will be shown if deployment windows aren't configured.
+
+You may click on any of them to view the details of the deployment windows.
+
+---
+
+## Result
+
+The below functions are blocked during an ongoing blackout window or outside maintenance window.
+
+* [Hibernation](#hibernation)
+* [Restart Workloads](#restart-workloads)
+* [Deployment](#deployment)
+* [Deletion of CD Pipeline](#deletion-of-cd-pipeline)
+
+{% hint style="info" %}
+However, the exempted users specified in deployment window configuration can bypass the restrictions and perform the above actions.
+{% endhint %}
+
+
+### Hibernation
```

Comment:
Definition of hibernation shouldn't be here, probably we can point user to right place

Reply:
Same is true for other objects mentioned below

Reply:
Remove the definitions from here and raise a new PR to cover the missing items (scale pods down to zero, restart workloads, etc in App doc), @ashokdevtron